data/source.py

```diff
@@ -0,0 +1,20 @@
+import csv
+
+import questions
+
+
+def from_csv_file(path: str) -> [questions.ExampleQuestion]:
+    with open(path, 'r', encoding='utf-8') as f:
+        return from_csv(f)
+
+
+def from_csv(iterable: any) -> [questions.ExampleQuestion]:
+    result = []
+    csv_reader = csv.reader(iterable)
+    next(csv_reader, None)  # escape headers is useless now
```

Comment:
Do not introduce time in a comment, correct description is:
is useless here

Reply:
got it.

Reply:
Where is the ”here”?
And the commit should be ”clarify comment: remove time info could outdated”

Reply:
QQ

Reply:
fixed.